web/views/dashboard.py

```diff
@@ -20,15 +23,26 @@ def dashboard(request):
     # Getting datasets related to the user
     last_datasets = Dataset.objects.filter(local_custodians=the_user).order_by('added')[:5]
 
-    # Get the notifications
-    # notifications = request.user.notifications.ordered()[:5]
+    # Return lists instead of QuerySets
+    return list(last_datasets), list(last_projects)
+
 
+def dashboard(request: HttpRequest) -> HttpResponse:
+    the_user = request.user
+
+    if the_user.is_anonymous():
```

Comment:
is_anonymous is not a method but attribute 

https://docs.djangoproject.com/en/4.0/ref/contrib/auth/#django.contrib.auth.models.User.is_anonymous

Reply:
my bad, I got misled by VSCode's hint; thanks!